### PR TITLE
Upgrade to Neutrino RC3

### PR DESCRIPTION
**Pull request by armenzg**

### CODE REVIEW

**Comment by armenzg** [comment]
@helfi92 I can't figure out what I need left for this to pass:
https://travis-ci.com/mozilla-frontend-infra/perf-goggles/builds/122860641

```
$ nyc -r lcov -r text yarn test

$ NODE_ENV=test mocha

/home/travis/build/mozilla-frontend-infra/perf-goggles/test/index.test.js:2

import fetchMock from 'fetch-mock';

^^^^^^
```

File: mocharc.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const neutrino = require('neutrino');
```
Comment (helfi92) [inline mocharc.js]: Is it possible this file needs to be renamed to `.mocharc.js`?

**Comment by armenzg** [comment]
Good catch on the name rename!

Now, I face a different issue:

```
Armens-MacBook-Pro:perf-goggles armenzg$ yarn test
yarn run v1.17.3
$ mocha
/Users/armenzg/repos/perf-goggles/node_modules/mocha/node_modules/yargs/yargs.js:1163
      else throw err
           ^

ReferenceError: regeneratorRuntime is not defined
```

**Comment by helfi92** [comment]
Have you tried refreshing `yarn.lock`?

**Comment by armenzg** [comment]
@helfi92 I think it is related that I test async code.
I had to use `import 'babel-polyfill';`:
https://github.com/babel/babel/issues/5085#issuecomment-277544677

Should something like this make it back to neutrino?
https://github.com/babel/babel/issues/5085#issuecomment-343369943

**Review by helfi92 — APPROVED** [review]
Answer to your question: https://github.com/neutrinojs/neutrino/issues/1279.

Code LGTM.

**Pull request merged**